File: gaurav2java/TakinginputArray.java
import java.util.*;

public class TakinginputArray {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter The size Of Array:");
        int n = sc.nextInt();
        int[] arr = new int[n];
        System.out.println("Enter The "+n+" Element");
        for (int i = 0; i < arr.length; i++) {
            arr[i] = sc.nextInt();
        }
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }
        sc.close();
    }
}

File: gaurav2java/Gaurav.html/Gaurav.c/takinginputArray.c
#include<stdio.h>
int main()
{
    int arr[5];
    for(int i = 0; i<=4; i++){
        int a = i+1;
        printf("Enter Element Number %d\n",a);
        scanf("%d",&arr[i]);
    }
    printf("Find the Array : %d",arr[2]);
    return 0;

}